src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/AbstractClassCouplingCheck.java

```diff
@@ -119,9 +125,19 @@ public final void setExcludedClasses(String... excludedClasses) {
             Collections.unmodifiableSet(Arrays.stream(excludedClasses).collect(Collectors.toSet()));
     }
 
+    /**
+     * Sets user-excluded pakcages to ignore. All exlcuded packages should end with a period,
+     * so it also appends a dot to a package name.
+     * @param excludedPackages the list of packages to ignore.
+     */
+    public final void setExcludedPackages(String... excludedPackages) {
+        this.excludedPackages = Collections.unmodifiableSet(Arrays.stream(excludedPackages)
+            .map(x -> x + DOT).collect(Collectors.toSet()));
```

Comment:
Thought about it, but @romani said, that the package name should be an identifier (https://github.com/checkstyle/checkstyle/issues/3309#issuecomment-284230528) and I removed checks and tests for the trailing dot. As for me, allowing user to specify a package name with a trailing dot or a star is a good option, but this is only *my* opinion. I could return a check for a dot, and mention it in the docs. Or, probably, we should add validation and show a warning to the user in case of invalid identifier.